scripts/ec2cleanup.py

```diff
@@ -72,26 +76,79 @@ def delete_unattached_volumes(provider_mgmt, excluded_volumes, output):
                                       headers=['Provider Key', 'Volume ID'],
                                       tablefmt='orgtbl'))
     except Exception as e:
-        logger.error(e)
+        print(e)
 
 
-def ec2cleanup(exclude_volumes, exclude_eips, output):
+def delete_unused_loadbalancers(provider_mgmt, excluded_elbs, output):
+    elb_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for elb in provider_mgmt.get_all_unused_loadbalancers():
+            if excluded_elbs and elb.name in excluded_elbs:
+                print "  Excluding Elastic LoadBalancer id: {}".format(elb.name)  # noqa
```

Comment:
@psav raised a point offline - these should be py3 compatible print statements.

At least update the modified ones here, if you'd like you can update all the prints in the script.